docs/operations/hotplug_interfaces.md

```diff
@@ -120,6 +127,31 @@ See the [Live Migration](./live_migration.md) docs for more details.
 
 Once the migration is completed the VM will have the new interface attached.
 
+>**Note**: This step can be done when both VM and VMI objects has the new interface in their spec.
+
+### Remove interface
+```bash
+virtctl removeinterface vm-fedora --name dyniface1
+```
+At this point the new interface should be detached from the guest but exist in the pod.
+
+### Migrate the VM
+```bash
+cat <<EOF kubectl apply -f -
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstanceMigration
+metadata:
+  name: migration-job
+spec:
+  vmiName: vmi-fedora
+EOF
+```
+See the [Live Migration](./live_migration.md) docs for more details.
+
+Once the VM is migrated, the interface will not exist in the migration target pod.
+
+>**Note**: This step can be done when the subject interface has `absent` state in both VM and VMI spec.
```

Comment:
Same, I suggest to note here in a more vague manner, this is too detailed and hard to follow.
Only someone that really understand well how things are working will be able to understand the note.

Reply:
what note do you suggest?

Reply:
https://github.com/kubevirt/user-guide/pull/690#discussion_r1241830282

>  It is recommended to avoid performing migrations in parallel to a hotplug. It is safer to assure the hotplug succeeded or at least reached the VMI specification before issuing a migration.

Reply:
I added the suggested note.